packages/exporter-trace-otlp-http/src/platform/node/util.ts

```diff
@@ -131,3 +132,12 @@ export function createHttpAgent(
     return undefined;
   }
 }
+
+export function configureCompression(compression: CompressionAlgorithm | undefined): CompressionAlgorithm {
+  if (compression) {
+    return compression;
+  } else {
+    const definedCompression = getEnv().OTEL_EXPORTER_OTLP_TRACES_COMPRESSION || getEnv().OTEL_EXPORTER_OTLP_COMPRESSION;
+    return definedCompression === 'gzip' ? CompressionAlgorithm.GZIP : CompressionAlgorithm.NONE;
```

Comment:
It seems like it is comparing with the enum values here. It would be great to not hardcode the value here.

```suggestion
    return definedCompression === CompressionAlgorithm.GZIP ? CompressionAlgorithm.GZIP : CompressionAlgorithm.NONE;
```